### ISSUE TITLE
Wrong position for first time in createPopup

### ISSUE THREAD

**Issue by Mizan-Rifat** [joeattardi/picmo#259]
When I click on my picker for the first time the popup appears at the left 0 position but after that it works fine. See my attachments...
![Screenshot 2023-01-31 at 4 38 26 PM](https://user-images.githubusercontent.com/26627242/215738444-40247ae1-419d-4844-9e02-f8076e30b07c.png)
![Screenshot 2023-01-31 at 4 38 36 PM](https://user-images.githubusercontent.com/26627242/215738575-9c25c676-58d3-4acc-b7d8-4f51512fbc1d.png)

**Comment by joeattardi** [comment]
Can you share a code sample that shows how you are opening the picker?

**Comment by Mizan-Rifat** [comment]
Here is my integration:

```
     const btn = document.querySelector('.emoji-btn');
     const picker = window.picmoPopup.createPopup(
        {},
        {
          referenceElement: btn,
          triggerElement: btn,
          position: 'bottom-start',
          showCloseButton: false
        }
      );
      btn.addEventListener('click', () => {
        picker.toggle();
      });`

**Comment by joeattardi** [comment]
Thank you! I see the issue now, I forgot to update the popup picker to accommodate the prefixed classes!

Once I made that change the positioning should work as expected, can you give the just-released 5.7.4 a try? 

Thanks.
